.eslintrc

```diff
@@ -6,7 +6,8 @@
 		"wpcalypso/react",
 		"plugin:jsx-a11y/recommended",
 		"prettier",
-		"plugin:@woocommerce/eslint-plugin/recommended"
+		"plugin:@woocommerce/eslint-plugin/recommended",
+		"plugin:import/typescript"
```

Comment:
Is this new dependency intentional?

Reply:
Somehow I ended up merging another branch inside of the one I was working. I've reverted the merge now